[TUTORIAL] Add Debifi

marshal-opsource opened the pull request:
Hello @LoicPandul 
This PR proposes more detailed integration of the debifi exchange.
closes #2256

marshal-opsource committed:
debifi1-exchange

LoicPandul committed:
Merge remote-tracking branch 'upstream/dev' into pr/3456

LoicPandul committed:
Update logo.webp

LoicPandul committed:
Update logo.webp

LoicPandul committed:
remove old tuto Debifi

LoicPandul committed:
Update tutorial.yml

LoicPandul committed:
update folder name

LoicPandul committed:
Update cover.webp

LoicPandul committed:
Update fr.md

LoicPandul committed:
Update fr.md

LoicPandul committed:
Update fr.md

LoicPandul committed:
fix img format

LoicPandul committed:
Update fr.md

LoicPandul reviewed (approved):
Very good tutorial, thank you for your contribution @marshal-opsource!

Here are the main changes I made on your branch:  
- I removed all the HTML code (mainly used on YouTube videos). Our platform doesn’t support HTML within a markdown document, except in very rare cases in courses. To embed YouTube videos, you can do it as I did: `![video](youtubelink)`. Like an image, but with a YouTube link as path.  
- I fixed the format of smartphone images (portraits) by putting them in a landscape rectangular frame, so they don’t appear oversized on the platform but are well-proportioned. I saw you used size indicators in the alt text of the image paths, but unfortunately our platform doesn’t support this type of parameter, so it needs to be done directly on the image itself.  

Apart from these details, your tutorial is very good, thank you!  

For the bounty payment, can I do it via Lightning? Could you please confirm your Lightning address? Thanks a lot!

marshal-opsource commented:
> Very good tutorial, thank you for your contribution @marshal-opsource!
> 
> Here are the main changes I made on your branch:
> 
> * I removed all the HTML code (mainly used on YouTube videos). Our platform doesn’t support HTML within a markdown document, except in very rare cases in courses. To embed YouTube videos, you can do it as I did: `![video](youtubelink)`. Like an image, but with a YouTube link as path.
> * I fixed the format of smartphone images (portraits) by putting them in a landscape rectangular frame, so they don’t appear oversized on the platform but are well-proportioned. I saw you used size indicators in the alt text of the image paths, but unfortunately our platform doesn’t support this type of parameter, so it needs to be done directly on the image itself.
> 
> Apart from these details, your tutorial is very good, thank you!
> 
> For the bounty payment, can I do it via Lightning? Could you please confirm your Lightning address? Thanks a lot!

I received the bounty. Thanks @LoicPandul 🙏
